<launch>
	<!-- Global params -->
	<rosparam param="image_resize_width">115</rosparam>
	<rosparam param="image_resize_height">44</rosparam>
	<rosparam param="patch_size">(9,9)</rosparam>

	<!-- Collect image data -->
	<node pkg="miro_teach_repeat" type="data_collect.py" name="data_collect" output="screen">
		<remap from="odom" to="odometry/filtered" />
		<remap from="image" to="ros_indigosdk_occam/image0" />
		<rosparam param="distance_threshold">0.2</rosparam>
		<rosparam param="angle_threshold_deg">15.0</rosparam>                           
	</node>

	<!-- Save Images and pose in a directory -->
	<node pkg="miro_teach_repeat" type="data_save.py" name="data_save" output="screen">
		<rosparam param="save_dir">~/robot/teach-repeat-data/</rosparam>
		<rosparam param="timestamp_folder">true</rosparam>
	</node>
</launch>

